api/src/test/java/io/opentelemetry/trace/TraceIdTest.java

```diff
@@ -50,9 +50,36 @@ public void isValid() {
   }
 
   @Test
-  public void getLowerLong() {
-    assertThat(first.getLowerLong()).isEqualTo(0);
-    assertThat(second.getLowerLong()).isEqualTo(-0xFF00000000000000L);
+  public void testGetRandomTracePart() {
+    byte[] id = {
+      0x01, 0x02, 0x03, 0x04, 0x05, 0x06, 0x07, 0x08, 0x09, 0x0A, 0x0B, 0x0C, 0x0D, 0x0E, 0x0F, 0x00
+    };
+    TraceId traceid = TraceId.fromBytes(id, 0);
```

Comment:
We also have the `TraceId` ctor taking two longs. Could we add a test for that one as well?

Reply:
Never mind, saw that Bogdan doesn't want us to use that ctor. Resolving.